app/controllers/projects/samples/metadata/fields_controller.rb

```diff
@@ -42,6 +42,34 @@ def update # rubocop:disable Metrics/AbcSize
           end
         end
 
+        def editable
```

Comment:
Need to add a rubocop disable to `class FieldsController < Projects::Samples::ApplicationController` above. Class has too many lines

Reply:
Added in 218ddb970c0b7334927c5c96189069d241243bfb